Simulator/Connections/Connections.cpp

```diff
@@ -44,6 +44,9 @@ Connections::Connections()
    function<void()> regGraphPropsFunc = bind(&Connections::registerGraphProperties, this);
    opsManager.registerOperation(Operations::registerGraphProperties, regGraphPropsFunc);
 
+   OperationManager::getInstance().registerOperation(Operations::registerGraphProperties,
+                                                     regGraphPropsFunc);
+
```

Comment:
You shouldn't need both of these calls. The first one is using `opsManager`, which is reference to the singleton returned by `OperationManager::getInstance().

In fact, I think this will result in `regGraphPropsFunc` being called twice by the operation manager.